test/base_test.py

```diff
@@ -129,6 +129,7 @@ def test_stats_on_array(self, server):
         image[0, 1] = np.inf
         finite_h5data = image[np.isfinite(image)]
         expected_stats = {
+            "positive_min": np.min(finite_h5data[finite_h5data > 0]),
             "min": np.min(finite_h5data),
             "max": np.max(finite_h5data),
             "mean": np.mean(finite_h5data),
```

Comment:
This test reimplements the code it's testing 😁 - might be better to create an array with known values and hard-code the positive min, min, max and mean. 🤷